src/clientapi/index.js

```diff
@@ -129,7 +129,8 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   var spritesheets;
   try {
     if (GLOBAL.CACHE) {
-      spritesheets = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+      var spriteSheetCache = GLOBAL.CACHE['spritesheets/map.json'] || GLOBAL.CACHE['spritesheets\\map.json'];
```

Comment:
I'm not too familiar with the build/import process for this project.  How would I go about importing path to the clientapi/index.js?  I've tried installing it via npm and requiring it via it's relative path.  

I end up with the following error message:
```
Error loading module:
     [[ ../../node_modules/path/path ]]
     requested by: modules/devkit-core/src/clientapi/index.js 
     current directory: http://localhost:9200/apps/pyly6l/ 
   Error: Could not import `...node_modules.path.path`
  	looked in:
  		modules/devkit-core/node_modules/path/path.js
  		modules/devkit-core/node_modules/path/path/index.js
```

I can verify that the file is indeed at `modules/devkit-core/node_modules/path/path.js` but that doesn't appear to be resolved.